scripts/package.json

```diff
@@ -37,21 +37,16 @@
     "prettier": "^2.7.1"
   },
   "dependencies": {
-    "@mongodb-js/connection-storage": "^0.7.3",
-    "@mongodb-js/devtools-docker-test-envs": "^1.2.5",
```

Comment:
Can we keep it here and re-expose the script you added so that you don't need to clone two repos and manually run commands from one in another?

Reply:
i thought through this one again and feel like that we should really keep the script in the docker repo. if we have to use the docker setup, we have to clone it to spin up the servers anyway. also since we are not using these connections on CI anywhere, it will be fine if a developer has to import connections from the docker repo instead of compass.  

I mostly want to avoid importing connections in Compass using a script which was original behaviour. If you strongly feel otherwise, we can then keep the current behaviour.

Reply:
No strong preference, I think it's okay to remove this stuff for now, it might be a fun skunkworks project to make an autoimport script for this 🙂